package.json

```diff
@@ -69,7 +69,8 @@
         "lodash.clonedeep": "~4.5.0",
         "most": "1.7.3",
         "most-subject": "5.3.0",
-        "sinek": "~9.0.0",
+        "node-rdkafka": "^2.10.1",
```

Comment:
@rob3000 probably the right thing to do until JS client api is fully supported one day

Reply:
Sounds good to me!